google_cloud.py

```diff
@@ -30,7 +26,10 @@ def sample_long_running_recognize(storage_uri):
     }
     audio = {"uri": storage_uri}
 
-    operation = client.long_running_recognize(config, audio)
+    try:
+        operation = client.long_running_recognize(config, audio)
+    except:
```

Comment:
naked excepts are not a good idea generally. is there any possibility of narrowing it down a bit, to expected errors?